views/js/e2e/_helpers/commands/setupCommands.js

```diff
@@ -124,5 +130,6 @@ Cypress.Commands.add('startTest', (testName) => {
         .contains(`Delivery of ${testName}`)
         .click();
 
-    cy.wait(['@testRunnerInit', '@testRunnerGetItem']);
+    cy.wait(['@testRunnerGetItem', '@testRunnerInit']);
```

Comment:
The order doesn't matter to Cypress (it's waiting for the responses, so requests made in the order `1,2` could return as `1,2` or `2,1` depending on network speed).

Reply:
I know @martijn-tao, but personally I prefer to put them in order.
Thanks for the warning anyway.

Reply:
That's fine. But isn't the usual order of the Test Runner `init` before `getItem`?